fetch_telem_main_script

```diff
@@ -11,4 +11,7 @@ sleep 7
 sleep 7
 /data/mta4/Script/SOH/fetch_telemetry.py -m flight
 sleep 7
-/data/mta4/Script/SOH/fetch_telemetry.py -m flight
\ No newline at end of file
+/data/mta4/Script/SOH/fetch_telemetry.py -m flight
+
+#Copy blobs to a backup location
+cp -f /data/mta4/www/CSH/blob_*.json /data/mta4/www/CSH/Backup/
```

Comment:
In the event that a JSON blob file is corrupted, these backups will be copied over from a previous telemetry fetch pass.

Reply:
What are the typical circumstances in which the json blob file can get corrupted?